src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/ProbationCaseEngagementService.kt

```diff
@@ -37,22 +35,22 @@ class ProbationCaseEngagementService(
   private fun handlePersonExistsForPnc(newOffenderDetail: DeliusOffenderDetail, person: PersonEntity) {
     val pnc = newOffenderDetail.identifiers.pnc!!
     val crn = newOffenderDetail.identifiers.crn
-    log.debug("Person record exist for pnc $pnc - adding new offender to person with crn $crn")
+    log.debug("Person record exists for pnc $pnc - adding new offender to person with crn $crn")
+    // should we not check that there is no offender first?
     addOffenderToPerson(person, createOffender(newOffenderDetail))
     trackEvent(TelemetryEventType.NEW_DELIUS_RECORD_PNC_MATCHED, crn, pnc)
   }
 
   private fun handleNoPersonForPnc(newOffenderDetail: DeliusOffenderDetail) {
     val pnc = newOffenderDetail.identifiers.pnc!!
     val crn = newOffenderDetail.identifiers.crn
-    log.debug("Person not exist for pnc $pnc - creating a new person and offender")
+    log.debug("Person record does not exist for pnc $pnc - creating a new person and offender")
     val newPerson = createNewPersonAndOffenderFromPnc(newOffenderDetail)
     trackEvent(TelemetryEventType.NEW_DELIUS_RECORD_NEW_PNC, crn, pnc, newPerson.personId.toString())
   }
 
-  private fun handleNoPnc(crn: String) {
+  private fun handleNoPnc() {
     log.debug("Pnc not present no further processing needed")
-    trackEvent(TelemetryEventType.NEW_DELIUS_RECORD_NO_PNC, crn)
```

Comment:
I do not think we want AppInsights for this either - we have the log if we need this info